wafer/static/js/edit_schedule.js

```diff
@@ -65,23 +61,20 @@
 
         e.target.classList.remove('over');
 
-        if (e.stopPropagation) {
-            e.stopPropagation(); // stops the browser from redirecting.
-        }
+        e.preventDefault(); // stops the browser from redirecting
 
         var slot = e.target.getAttribute('data-slot');
         var venue = e.target.getAttribute('data-venue');
 
         //noinspection JSUnresolvedVariable
```

Comment:
Can we remove this comment now?

Any idea how the `event.` references worked before? Some sort of crazy magic?